components/ntp_background_images/browser/ntp_sponsored_rich_media_source.h

```diff
@@ -23,7 +24,8 @@ class NTPBackgroundImagesService;
 
 class NTPSponsoredRichMediaSource : public content::URLDataSource {
  public:
-  explicit NTPSponsoredRichMediaSource(NTPBackgroundImagesService* service);
+  explicit NTPSponsoredRichMediaSource(NTPBackgroundImagesService* service,
```

Comment:
nit: explicit is required only for one argument

Reply:
Fixed, thanks